osu.Game.Rulesets.Osu/Mods/OsuModAutopilot.cs

```diff
@@ -36,21 +39,63 @@ public class OsuModAutopilot : Mod, IApplicableFailOverride, IUpdatableByPlayfie
 
         private int currentFrame;
 
+        int cachedFrames = 1;
+        private const float relax_leniency = 3f;
```

Comment:
there cannot be references to "relax" in autopilot code.

Reply:
Ok, a better name would be.... offset?